dronecan/sensors/rc/1141.RCLinkStats.uavcan

```diff
@@ -0,0 +1,39 @@
+#
+# This definition describes metadata and statistics for a RF control link.
+# It provides information regarding a link's signal quality, as well as transmitter
+# power, and antenna information. It accomodates single-radio, and dual-radio
+# link architectures. It provides information for both uplink (control unit to airborne
+# receiver), and downlink (airborne receiver to control unit) data.
+#
+
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+
+# Uplink - received signal strength, in decibel-milliwatts (RSSI dBm) for antennas 1 and 2 respectively. 
+# Ranges from # -255 to 0. (An integer value of 100 means RSSI is -100dBm)
+uint8 uplink_rssi_1
+uint8 uplink_rssi_2
+
+# Link quality, on a scale from 1 to 255. Is a proxy for packet loss. 0 means data is unavailable. 1 and 255 are mapped to 0% and 100% of packets received.
+uint8 uplink_link_quality
```

Comment:
This version would align with MAVLink better. 
- MAVLink uses 0 as a proxy for invalid by preference, and max field values if "0" is meaningful number for the field., In this case 0 is valid, so suggest we swap around.
- Usually it is easier for users parsing the data to supply percentage data as a 1..100 range. The only reason to use the full range is if you need finer resolution, which I do not believe is the case here.
- Have no discussed the words with OllieW yet.

```suggestion
# Link quality, as a percentage of valid packets received over some time period [1..100]. 256 means data is unavailable.
uint8 uplink_link_quality
```